GUI/TASWindow.py

```diff
@@ -41,6 +42,7 @@ def update(self: Self, delta: float) -> None:
                 time_of_day_manager_handle().update()
                 level_up_manager_handle().update()
                 combat_manager_handle().update()
+                shop_manager_handle().update()
```

Comment:
Maybe put this one under the timer below. It doesn't need to be updated every frame probably?